### PR TITLE
Use the correct verifier for DI VCs

### LINKED ISSUE #612: [Bug] Verification of valid DID Configuration fails
**Describe the bug**
On a local running instance, running the following
```shell
curl -X PUT 'localhost:3000/v1/did-configurations/verification' -d '{
  "origin": "https://identify.foundation"
}'
```

returns 
```json
{
  "verified": false,
  "didConfiguration": "{\n  \"@context\": \"https://identity.foundation/.well-known/did-configuration/v1\",\n  \"linked_dids\": [\n    {\n      \"@context\": [\n        \"https://www.w3.org/2018/credentials/v1\",\n        \"https://identity.foundation/.well-known/did-configuration/v1\"\n      ],\n      \"issuer\": \"did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM\",\n      \"issuanceDate\": \"2020-12-04T14:08:28-06:00\",\n      \"expirationDate\": \"2025-12-04T14:08:28-06:00\",\n      \"type\": [\"VerifiableCredential\", \"DomainLinkageCredential\"],\n      \"credentialSubject\": {\n        \"id\": \"did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM\",\n        \"origin\": \"https://identity.foundation\"\n      },\n      \"proof\": {\n        \"type\": \"Ed25519Signature2018\",\n        \"created\": \"2020-12-04T20:08:28.540Z\",\n        \"jws\": \"eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..D0eDhglCMEjxDV9f_SNxsuU-r3ZB9GR4vaM9TYbyV7yzs1WfdUyYO8rFZdedHbwQafYy8YOpJ1iJlkSmB4JaDQ\",\n        \"proofPurpose\": \"assertionMethod\",\n        \"verificationMethod\": \"did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM#z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM\"\n      }\n    },\n    \"eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa29USHNnTk5yYnk4SnpDTlExaVJMeVc1UVE2UjhYdXU2QUE4aWdHck1WUFVNI3o2TWtvVEhzZ05OcmJ5OEp6Q05RMWlSTHlXNVFRNlI4WHV1NkFBOGlnR3JNVlBVTSJ9.eyJleHAiOjE3NjQ4NzkxMzksImlzcyI6ImRpZDprZXk6ejZNa29USHNnTk5yYnk4SnpDTlExaVJMeVc1UVE2UjhYdXU2QUE4aWdHck1WUFVNIiwibmJmIjoxNjA3MTEyNzM5LCJzdWIiOiJkaWQ6a2V5Ono2TWtvVEhzZ05OcmJ5OEp6Q05RMWlSTHlXNVFRNlI4WHV1NkFBOGlnR3JNVlBVTSIsInZjIjp7IkBjb250ZXh0IjpbImh0dHBzOi8vd3d3LnczLm9yZy8yMDE4L2NyZWRlbnRpYWxzL3YxIiwiaHR0cHM6Ly9pZGVudGl0eS5mb3VuZGF0aW9uLy53ZWxsLWtub3duL2RpZC1jb25maWd1cmF0aW9uL3YxIl0sImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImlkIjoiZGlkOmtleTp6Nk1rb1RIc2dOTnJieThKekNOUTFpUkx5VzVRUTZSOFh1dTZBQThpZ0dyTVZQVU0iLCJvcmlnaW4iOiJpZGVudGl0eS5mb3VuZGF0aW9uIn0sImV4cGlyYXRpb25EYXRlIjoiMjAyNS0xMi0wNFQxNDoxMjoxOS0wNjowMCIsImlzc3VhbmNlRGF0ZSI6IjIwMjAtMTItMDRUMTQ6MTI6MTktMDY6MDAiLCJpc3N1ZXIiOiJkaWQ6a2V5Ono2TWtvVEhzZ05OcmJ5OEp6Q05RMWlSTHlXNVFRNlI4WHV1NkFBOGlnR3JNVlBVTSIsInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJEb21haW5MaW5rYWdlQ3JlZGVudGlhbCJdfX0.aUFNReA4R5rcX_oYm3sPXqWtso_gjPHnWZsB6pWcGv6m3K8-4JIAvFov3ZTM8HxPOrOL17Qf4vBFdY9oK0HeCQ\"\n  ]\n}\n",
  "reason": "could not create validator for kid did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM#z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM: could not convert private key to JWK: did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM#z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM: unsupported private key type: ed25519.PublicKey"
}
```

**Expected behavior**
`verified` should be `true`

### PR DESCRIPTION
**Pull request by andresuribe87**
# Overview
Fixes #612 by using the verifier that only has the public key.

# Description
This was a simple oversight, as `NewDataIntegrityKeyAccess` does not accept a public key as the third parameter.

# How Has This Been Tested?
Added a test the failed before, and passes after. The mock crazyness is all so that json-ld works without having to retrieve things from the network.

### CODE REVIEW

File: pkg/server/server_did_configuration_test.go [hunk 1]
```diff
@@ -20,6 +20,340 @@ import (
 	"gopkg.in/h2non/gock.v1"
 )
 
+const w3cCredentialContext = `{
```
Comment (decentralgabe) [inline pkg/server/server_did_configuration_test.go:23]: can this be loaded from a file instead? I thought we had context caching somewhere else...

**Review by decentralgabe — CHANGES_REQUESTED** [review]

File: pkg/server/server_did_configuration_test.go (hunk 1 above)
Comment (andresuribe87) [reply pkg/server/server_did_configuration_test.go:23]: I wasn't aware. I found https://github.com/TBD54566975/ssi-sdk/pull/102/files , but we need to do some more piping in order to have the cache be pre-loaded. Right now, the processor caches documents, but it still needs to fetch them once from the network. 

Ok if I take on a follow up? Requires changes in SDK and the service.

Comment (decentralgabe) [reply pkg/server/server_did_configuration_test.go:23]: certainly, in the meantime can you move this to a file?

**Review by decentralgabe — APPROVED** [review]

File: pkg/server/server_did_configuration_test.go (hunk 1 above)
Comment (andresuribe87) [reply pkg/server/server_did_configuration_test.go:23]: Tracking in https://github.com/TBD54566975/ssi-service/issues/619

**Pull request merged**